src/components/InventoryTable/EntityTableToolbar.js

```diff
@@ -194,6 +200,10 @@ const EntityTableToolbar = ({
 
     const shouldReload = page && perPage && filters && (!hasItems || items);
 
+    useEffect(() => {
+        dispatch(fetchOperatingSystems());
```

Comment:
Not needed for now, but I'd like to see a followup PR with adding applied filters when fetching OSs, that way users won't see an OS which is not available for current filter

Reply:
@karelhala, could you please elaborate more on that (maybe, provide some examples)? This change is a main goal of this PR, and it was requested by UX to always show the list of OS versions available for the account. Another note from them also was that the list shouldn't change with other filters being applied.

Reply:
If the list shouldn't change with other filters applied I think we are good to go with this. The API should return just the OS versions user can apply.

Reply:
@katierik Just tagging you for awareness

Reply:
Yes, exactly :) operating_system should respond only with the versions that are available for the current account.